code/game/gamemodes/changeling/powers/mutations.dm

```diff
@@ -264,8 +264,9 @@
 
 /mob/proc/tentacle_grab(mob/living/carbon/C)
 	if(Adjacent(C))
-		var/obj/item/grab/G = C.grabbedby(src,1)
```

Comment:
This should work and if it doesn't, it's a sign of a larger issue. grabbedby should always return the grab when it is created, and if it doesn't, it should be a clear sign that the grab does not actually exist. If this is not the case, then something is currently wrong with grab code and that problem should be resolved rather than blindly patching over it.

Reply:
Agreed. Duct-tape solutions aren't wanted.

Reply:
How's this now? I found a missing return in human grab